config/initializers/assets.rb

```diff
@@ -17,7 +17,7 @@
 # folder are already added.
 Rails.application.config.assets.precompile += %w(
   fullCalendar.js fullCalendar.css
-  openStreetMaps.js
+  openStreetMaps.js documents.js
```

Comment:
put the new js file declaration on its own line (every file here are kinda classified by their scope